source/client.c

```diff
@@ -71,6 +77,9 @@ static void s_mqtt_client_shutdown(
 
     struct aws_mqtt_client_connection *connection = user_data;
 
+    AWS_LOGF_TRACE(
+        AWS_LS_MQTT_CLIENT, "id=%p Channel has been shutdown with error code %d", (void *)connection, error_code);
```

Comment:
Other systems put a colon after the identifiers like "id=%p: Content of message"